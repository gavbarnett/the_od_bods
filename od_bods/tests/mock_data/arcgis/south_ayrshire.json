{
    "data": [],
    "meta": {
        "queryParameters": {
            "catalog": {
                "groupIds": "any(436655c931664e279f675390213d828e)"
            },
            "filter": {
                "isCancelled": "eventsEq(false)",
                "status": "not(private,draft,planned)",
                "endDate": "eventsBefore(1663502679168)"
            }
        },
        "stats": {
            "count": 0,
            "totalCount": 0,
            "aggs": {}
        }
    }
}