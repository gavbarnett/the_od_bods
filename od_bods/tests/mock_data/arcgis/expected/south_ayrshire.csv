Title,Owner,PageURL,AssetURL,FileName,DateCreated,DateUpdated,FileSize,FileSizeUnit,FileType,NumRecords,OriginalTags,ManualTags,License,Description
